robot_interface:
  ros__parameters:
    robot_yaml_path: "/home/hidro/odri_ws/src/odri_ros2/odri_examples/config/robots/stretch_legs.yaml"
    adapter_name: "enp61s0"
    n_slaves: 1
